TestAssets/TestProjects/TestMscorlibReference/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {},
+  "frameworks": {
+    "netstandardapp1.5": {
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc2-23931"
```

Comment:
Good catch. @troydai make these consistent with whatever everything else is using.